frontend/src/components/contexts/contexts.ts

```diff
@@ -1,14 +1,13 @@
 import * as React from "react";
 import { AuthenticationHandlerService } from "../../services/authentication_handler_service/authentication_handler_service";
-import { LoginStatusHandlerService } from "../../services/login_status_handler_service/login_status_handler_service";
 import { ProfileHandlerService } from "../../services/profile_handler_service/profile_handler_service";
 import { YourClubsHandlerService } from "../../services/your_clubs_handler_service/your_clubs_handler_service";
 
 interface ServicesInterfaces {
   profileHandlerService: ProfileHandlerService,
   yourClubsHandlerService: YourClubsHandlerService,
   authenticationHandlerService: AuthenticationHandlerService,
-  loginStatusHandlerService: LoginStatusHandlerService,
 }
 
-export const ServiceContext = React.createContext<undefined|ServicesInterfaces>(undefined);
+export const ServiceContext =
+    React.createContext<undefined | ServicesInterfaces>(undefined);
```

Comment:
super nit: I'm not quite sure what the TS styling convention is, but I haven't put a space between my types in my PRs so far and haven't been told the styling was incorrect. We should choose either spaces or no spaces for consistency.

Reply:
TypeScript documentation uses a space between the `|` (see https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types) and thus I deferred to their style. I have no strong preference though, I can go through the code and switch to either one.

Reply:
I've seen both ways. In general we do put spaces around all operators but unions are sort of a special case, so some people don't. I usually do in my code but, I don't really care either way. Style guides don't cover everything

Reply:
After talking to Jimmy and consulting https://github.com/google/gts we've decided to put spaces around the `|`, I've changed the code to be consistent and pushed the result.